source/about/cloud-subscriptions.rst

```diff
@@ -46,9 +46,9 @@ Frequently asked questions
 What happens when my 14-day trial period ends?
```

Comment:
When Cloud Freemium is released, is there going to be a 14-day trial period?

Reply:
Good question. I'll need to check but for now let's leave this as-is and I have noted this down as needing an update/review for 7.0 docs.